### PR TITLE
Address a few issues in Transform3D documentation

### PR DESCRIPTION
**Pull request by Mickeon**
Self-explanatory (see latest comments in https://github.com/godotengine/godot/pull/87334).

### CODE REVIEW

**Review by AThousandShips — APPROVED** [review]

File: doc/classes/Transform3D.xml [hunk 1]
```diff
@@ -247,28 +247,28 @@
 			<return type="Transform3D" />
 			<param index="0" name="right" type="float" />
 			<description>
-				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale uniformly, also resizing the [member basis].
+				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale uniformly, scaling the [member basis].
```
Comment (fire) [inline doc/classes/Transform3D.xml:250]:
```suggestion
				Multiplies all components of the [Transform3D] by the given [float], including the [member origin]. This affects the transform's scale uniformly, and scaling the [member basis].
```

This happens around 4 times. Isn't this a list?

Comment (AThousandShips) [reply doc/classes/Transform3D.xml:250]: It isn't a list, it's a sub clause

Comment (fire) [reply doc/classes/Transform3D.xml:250]: Sounds good.

Comment (Mickeon) [reply doc/classes/Transform3D.xml:250]: I can reword the description outright if it's giving trouble. I just want to make it clear that the whole transform (`origin` **AND** `basis`) are affected.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 4.2.2.
Cherry-picked for 4.1.4.

**Pull request merged**